modules/log_forwarder/main.tf

```
@@ -24,7 +24,7 @@ data "aws_region" "current" {}
 
 module "this_s3_bucket" {
   source  = "terraform-aws-modules/s3-bucket/aws"
-  version = "v3.0.1"
+  version = "v3.3.1"
```

```suggestion
  version = "v3.6.1"
```
I think we can safely upgrade to the latest version of this module.